app/controllers/staff/activity_forms_controller.rb

```diff
@@ -76,6 +77,8 @@ def show
       skip_step if @activity.fund?
     when :oda_eligibility_lead
       skip_step unless @activity.is_project?
+    when :uk_dp_named_contact
+      skip_step unless @activity.is_project? && @activity.is_newton_funded?
```

Comment:
I don't think we need `is_project?` if we have `is_newton_funded?` do we?

Reply:
Programmes can also be Newton funded, and they sit above projects.